src/components/common/Modal/ConfirmModal.tsx

```
@@ -2,17 +2,28 @@ import styled from '@emotion/styled';
+  confirmText = '예',
+  cancelText = '아니요',
+}: ConfirmModalProps) => {
+  const handleContainerClick = (e: React.MouseEvent) => {
+    e.stopPropagation();
+  };
+
   return (
-    <Container>
+    <Container onClick={handleContainerClick}>
       <div className="title">{title}</div>
       <div className="disc" dangerouslySetInnerHTML={{ __html: disc }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)